MSAL/src/native_auth/network/responses/MSALNativeAuthResendCodeRequestResponse.swift

```diff
@@ -27,9 +27,9 @@ import Foundation
 struct MSALNativeAuthResendCodeRequestResponse: Decodable {
 
     // MARK: - Variables
-    let credentialToken: String
+    let continuationToken: String
 
     enum CodingKeys: String, CodingKey {
-        case credentialToken = "flowToken"
+        case continuationToken = "continuation_token"
```

Comment:
This change is not needed. You can remove `CodingKeys`

Reply:
Removed